### PR TITLE
DED-533 tagging v2 Conf as deprecated and hiding it

### PR DESCRIPTION
**Pull request by kamilatokarska**
# Deploy preview

https://63f6029cb1318a7af9c98915--livechat-public-docs.netlify.app/management/configuration-api/v2.0

# Links

Link your Jira ticket.

- [Jira]()

# Description

# Review and release

Apply changes and resolve conflicts. Once the described functionality lands on prod, let us know on #platform-docs-releases that your PR is ready for release. We will **merge** and **publish** the changes.

### CODE REVIEW

File: src/pages/management/configuration-api/v2.0/index.mdx [hunk 1]
```diff
@@ -14,7 +14,7 @@ Welcome to the LiveChat API documentation!
 
 Our API provides a set of flexible tools which you can use to create new outstanding projects. We smile a bit more each time we see skilled developers unleash their creativity!
```
Comment (ale-kacperczyk) [inline src/pages/management/configuration-api/v2.0/index.mdx:15]: We discussed that it'd be good to include a visible warning/emoji with information about this version being deprecated

Comment (kamilatokarska) [reply src/pages/management/configuration-api/v2.0/index.mdx:15]: Okay, I wrote there that it is deprecated, but a warning would definitely be better. I must have misunderstood something. I'll fix it.

File: src/pages/management/configuration-api/index.mdx [hunk 2]
```diff
@@ -3067,7 +3067,7 @@ curl -X POST \
 
 Enables the webhooks [registered](#register-webhook) by a given **Client Id** (application). The webhooks will be installed for the license associated with the access token used in the request.
 
-Webhooks will be automaticall enabled if you [register them via Developer Console](/getting-started/app-guides/chat-webhooks/).
```
Comment (kamilatokarska) [inline src/pages/management/configuration-api/index.mdx:3070]: fixed typo from #platform-docs (haven't updated preview for that change as it's tiny)

File: src/pages/management/configuration-api/v3.4/index.mdx [hunk 3]
```diff
@@ -2731,7 +2731,7 @@ curl -X POST \
 
 Enables the webhooks [registered](#register-webhook) by a given **Client Id** (application). The webhooks will be installed for the license associated with the access token used in the request.
 
-Webhooks will be automaticall enabled if you [register them via Developer Console](/getting-started/app-guides/chat-webhooks/).
```
Comment (kamilatokarska) [inline src/pages/management/configuration-api/v3.4/index.mdx:2734]: fixed typo from #platform-docs (haven't updated preview for that change as it's tiny)

File: src/pages/management/configuration-api/v3.6/index.mdx [hunk 4]
```diff
@@ -3057,7 +3057,7 @@ curl -X POST \
 
 Enables the webhooks [registered](#register-webhook) by a given **Client Id** (application). The webhooks will be installed for the license associated with the access token used in the request.
 
-Webhooks will be automaticall enabled if you [register them via Developer Console](/getting-started/app-guides/chat-webhooks/).
```
Comment (kamilatokarska) [inline src/pages/management/configuration-api/v3.6/index.mdx:3060]: fixed typo from #platform-docs (haven't updated preview for that change as it's tiny)

File: src/pages/management/configuration-api/v2.0/index.mdx (hunk 1 above)
Comment (kamilatokarska) [reply src/pages/management/configuration-api/v2.0/index.mdx:15]: Corrected :) I also added the info about 30 days

File: src/pages/management/configuration-api/v2.0/index.mdx [hunk 5]
```diff
@@ -14,7 +14,11 @@ Welcome to the LiveChat API documentation!
...
 
-Please note that this documentation refers to the **API version 2.0**. This is the **legacy** version. We encourage you to migrate to the latest stable version. [Read more about versioning...](/management/#api-versioning)
+<Warning>
+
+Please note that this documentation refers to the **API version 2.0**. This is the **deprecated** version, and you'll maintain access only to methods used in the last 30 days. We encourage you to migrate to the latest stable version. [Read more about versioning...](/management/#api-versioning)
```
Comment (ale-kacperczyk) [inline src/pages/management/configuration-api/v2.0/index.mdx:19]: When someone enters the documentation later than on the day of closing v2, the "last 30 days" will no apply. You know what I mean? I wouldn't mention it I guess

Comment (kamilatokarska) [reply src/pages/management/configuration-api/v2.0/index.mdx:19]: Okay, I get it. You're right. Haven't thought about it this way. I'll remove it

Comment (kamilatokarska) [inline src/pages/management/configuration-api/v2.0/index.mdx:19]:
```suggestion
Please note that this documentation refers to the **API version 2.0**. This is the **deprecated** version. We encourage you to migrate to the latest stable version. [Read more about versioning...](/management/#api-versioning)
```

**Review by ale-kacperczyk — APPROVED** [review]

**Pull request merged**